### PR TITLE
Add promo code text to i18n

### PR DESCRIPTION
**Pull request by chaotixkilla**
This branch was rebased with `fix-promo-codes-ui`, in order to add the recent changes (changing everything related to `coupons` to `promo codes`). Once that branch is merged with dev, this one will need to be rebased, as to clean the commit log

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Add new screen text to i18n](https://trello.com/c/F96BxfTs/540-add-new-screen-text-to-i18n)

**Review by josepfo — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

File: i18n/ui/lbm-ui_en-US.po [hunk 1]
```diff
@@ -442,9 +474,17 @@ msgstr "In order to proceed, we are required to verify your identity."
...
+msgid "Please press the front of your state ID card against the scan window, hold it there, and then press "Ready to scan"."
+msgstr "Please press the front of your state ID card against the scan window, hold it there, and then press "Ready to scan"."
+
+#: On screen: scan-photo-manual
+msgid "Ready to scan"
+msgstr "Ready to scan"
+
 #: On screen: scan-photo
-msgid "Please press the front of your <strong>state ID card</strong> against the scan window."
-msgstr "Please press the front of your <strong>state ID card</strong> against the scan window."
```
Comment (joshmh) [inline i18n/ui/lbm-ui_en-US.po:447]: Why are removing this tag?

File: i18n/ui/lbm-ui_en-US.po [hunk 2]
```diff
@@ -647,8 +687,8 @@ msgid "We see your transaction but we're still waiting for confirmation. This ta
 msgstr "We see your transaction but we're still waiting for confirmation. This takes 10 minutes on average but can take up to an hour or two."
 
 #: On screen: unconfirmed-deposit
-msgid "Please come back for your cash and press <strong>Redeem</strong> when you get our text message."
-msgstr "Please come back for your cash and press <strong>Redeem</strong> when you get our text message."
```
Comment (joshmh) [inline i18n/ui/lbm-ui_en-US.po:651]: Why are we removing this tag?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: i18n/ui/lbm-ui_en-US.po (hunk 2 above)
Comment (RafaelTaranto) [reply i18n/ui/lbm-ui_en-US.po:651]: The tag doesn't really work with the translation tools.

**Pull request merged**